app/global/Rfc1123DateTimeParser.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright 2016 Groupon.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package global;
+
+import org.joda.time.DateTime;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+
+import java.util.Locale;
+
+/**
+ * A parser and stringifier for the 'http datetime format' specified in RFC 822, updated by RFC 1123.
+ *
+ * @author Matthew Hayter (mhayter at groupon dot com)
+ */
+public class Rfc1123DateTimeParser {
+
+    private static final DateTimeFormatter RFC_1123_DATE_TIME_FORMATTER =
+            DateTimeFormat.forPattern("EEE, dd MMM yyyy HH:mm:ss 'GMT'").withZoneUTC().withLocale(Locale.US);
```

Comment:
:rage4: 

I think my problem was that I was looking for a joda formatter and skipped right over the Java8 native one. Does this project prefer one or the other? Seems to be joda...?